test/test_kml_generator.py

```diff
@@ -26,3 +29,7 @@ def test_create_linestrings(self):
         location_list_2 = [(Location(54.54, -1.45, 150, "point1_2")), (Location(54.54, -1.44, 150, "point2_2"))]
         linestring_list = create_linestrings(location_list, location_list_2, line_of_sight_folder)
         self.assertEqual("point1, point1_2", str(linestring_list[0].name))
+
+    @staticmethod
+    def create_test_file_path():
+        return os.path.join(CURRENT_DIR, os.pardir, "test/data/")
```

Comment:
Previously we were referring to test files by just `csv/filename.csv`, which only works if you're running from the `test` folder. We want to be able to run our test classes from any location and have them figure out where the data files are. This method finds the path to the running file (so `D:/Ben/Git/line-of-sight-analysis/test/test_kml_generator.py`) and then navigates to the data directory relative to that (by sticking `data/` to the end of it).